airflow/www/static/js/dag/details/taskInstance/Logs/utils.ts

```diff
@@ -47,11 +47,15 @@ export const parseLogs = (
     return {};
   }
   let lines;
+
+  let warning;
+
   try {
     lines = data.split('\n');
   } catch (err) {
     console.error(err);
-    return {};
+    warning = 'Unable to show logs. There was an error parsing logs.';
```

Comment:
We might remove the `console.error` just above. eslint is complaining about it and now that the error is properly handled it's not required anymore.